### PR TITLE
chore(select): add example of empty menu with a div

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/2333

### CODE REVIEW

File: src/patternfly/components/Select/select-single.hbs [hunk 1]
```diff
@@ -1,9 +1,12 @@
-<ul class="pf-c-select__menu{{#if select-menu--modifier}} {{select-menu--modifier}}{{/if}}"
+<{{#if select-menu--type}}{{select-menu--type}}{{else}}ul{{/if}} class="pf-c-select__menu{{#if select-menu--modifier}} {{select-menu--modifier}}{{/if}}"
   aria-labelledby="{{id}}-label"
   {{#unless select--IsExpanded}}hidden{{/unless}}
   {{#if select--attribute}}
     {{{select--attribute}}}
   {{/if}}>
+  {{#if select--IsEmptyMenu}}
```
Comment (christiemolloy) [inline src/patternfly/components/Select/select-single.hbs]: should this be `select--HasEmptyMenu` ?

Comment (mcoker) [reply src/patternfly/components/Select/select-single.hbs]: I dunno... I think `Is[whatever]` seems to show an example variation. Like `IsExpanded`, `IsTypeahead`, or `IsMultiselect` identifies the example. `IsEmptyMenu` shows the empty menu example. I see `HasGroups` or `HasCustomLabel` more as a toggle to include something that is optional, though the line seems blurry and I could totally go either way with it :) Let me know if you'd still like me to update it.

**Comment by mcoker** [comment]
@christiemolloy rebased and updated since this was using the old hbs format

File: src/patternfly/components/Select/select-single.hbs (hunk 1 above)
Comment (christiemolloy) [reply src/patternfly/components/Select/select-single.hbs]: Thanks for the explanation that definitely makes sense from the example side. "Is Empty Menu" just reads strange, but no worries!

**Comment by christiemolloy** [comment]
Approved but there is a conflict

**Pull request merged**